servicetalk-concurrent-api-test/src/test/java/io/servicetalk/concurrent/api/test/PublisherStepVerifierTest.java

```diff
@@ -729,19 +728,14 @@ private static void verifyException(Supplier<Duration> verifier, String failedTe
     }
 
     static void verifyException(Supplier<Duration> verifier, String classNamePrefix, String failedTestMethod) {
-        try {
-            verifier.get();
-        } catch (StepAssertionError error) {
-            StackTraceElement[] stackTraceElements = error.getStackTrace();
-            assertThat(stackTraceElements.length, greaterThanOrEqualTo(1));
-            assertThat("first stacktrace element expected <class: " + classNamePrefix +
-                    "> actual: " + stackTraceElements[0] + " error: " + error,
-                    stackTraceElements[0].getClassName(), startsWith(classNamePrefix));
-            StackTraceElement testMethodStackTrace = error.testMethodStackTrace();
-            assertEquals("unexpected test method failure: " + testMethodStackTrace, failedTestMethod,
-                    testMethodStackTrace.getMethodName());
-            return;
-        }
-        fail();
+        StepAssertionError error = assertThrows(StepAssertionError.class, verifier::get);
+        StackTraceElement[] stackTraceElements = error.getStackTrace();
+        assertThat(stackTraceElements.length, greaterThanOrEqualTo(1));
+        assertThat("first stacktrace element expected <class: " + classNamePrefix +
+                        "> actual: " + stackTraceElements[0] + " error: " + error,
```

Comment:
nit: consider putting more content on the previous line now that indentation has changed.